addons/medical/functions/fnc_fullHeal.sqf

```diff
@@ -0,0 +1,26 @@
+#include "..\script_component.hpp"
+/*
+ * Author: DartRuffian
+ * Fully heals the patient.
+ *
+ * Arguments:
+ * 0: Medic <OBJECT>
+ * 1: Patient <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, cursorObject] call ace_medical_fnc_fullHeal
+ *
+ * Public: Yes
+ */
+
+params [["_medic", objNull, [objNull]], ["_patient", objNull, [objNull]]];
+
+if (isNull _medic or !alive _patient) exitWith {
```

Comment:
Should dead medics be allowed to heal?

I couldn't tell if a dead medic would cause any issues from a quick look, but I'm not experienced with medical's internals

Reply:
Or alternatively, maybe the parameters should be swapped and _medic be made optional, so you could just do `[_unit] call ace_medical_fnc_fullHeal` in cases where there is no medic?